apps/src/templates/sectionAssessments/StudentAssessmentOverviewTable.jsx

```diff
@@ -0,0 +1,174 @@
+import React, {Component, PropTypes} from 'react';
+import {Table, sort} from 'reactabular';
+import {tableLayoutStyles, sortableOptions} from "../tables/tableConstants";
+import commonMsg from '@cdo/locale';
+import wrappedSortable from '../tables/wrapped_sortable';
+import orderBy from 'lodash/orderBy';
+import MultipleChoiceAnswerCell from './MultipleChoiceAnswerCell';
+
+export const COLUMNS = {
+  QUESTION: 0,
+  STUDENT_ANSWER: 1,
+  CORRECT_ANSWER: 2,
+};
+
+const studentAnswerDataPropType = PropTypes.shape({
```

Comment:
These shape definitions should only be defined once, and then imported into each file you need. You could have file called `assessmentDataShapes.js` to do this.

Reply:
Fixed.  Thanks